plugins/generator-1.16.5/forge-1.16.5/utils/procedures.java.ftl

```diff
@@ -62,6 +62,14 @@
     </#if>
 </#macro>
 
+<#macro procedureOBJToItemstackCode object="">
+    <#if object?? && object?has_content && object.getName() != "null">

```

Comment:
If you return itemstack, you need to add @ItemStack designator for mapping system

Reply:
Could you show me an example plz?

Reply:
Check this file https://github.com/MCreator/MCreator/blob/7c85adef256c675ed51b165c29b4533b649e20d7/plugins/generator-1.16.5/forge-1.16.5/variables/itemstack.yaml#L5

Reply:
You can also check this method in `.../utils/mcitems.ftl`. https://github.com/MCreator/MCreator/blob/7c85adef256c675ed51b165c29b4533b649e20d7/plugins/generator-1.16.5/forge-1.16.5/utils/mcitems.ftl#L17 It's why it's needed.